tests/helpers.py

```diff
@@ -371,9 +372,52 @@ def wait_switches_connect(self):
                 status = [(sw.name, sw.connected()) for sw in self.net.switches]
                 raise Exception('Timeout: timed out waiting switches reconnect. Status %s' % status)
 
+    def wait_kytos_links(self):
+        wait_count = 0
+        last_error = ""
+        topo_links = []
+        for link in self.net.links:
+            if link.intf1.node in self.net.switches and link.intf2.node in self.net.switches:
+                link_id = self.create_link_id(link)
+                if link_id:
+                    topo_links.append(link_id)
+        while wait_count < 30:
+            try:
+                response = requests.get("http://127.0.0.1:8181/api/kytos/topology/v3/links/", timeout=3)
+                links = response.json()["links"]
+                assert len(topo_links) == len(links), f"{topo_links=} {links=}"
+                assert all(link_id in links for link_id in topo_links), f"{topo_links=} {links=}"
```

Comment:
In the future, it might be useful to also parametrize if it should also assert that all links are UP too, wdyt? This could also help us to avoid setting larger time sleeps in other subsequent parts